examples/graphics/area-lights.html

```diff
@@ -0,0 +1,269 @@
+    <script src="../../build/playcanvas-extras.js"></script>
+    <style>
+        body { 
+            margin: 0;
+            overflow-y: hidden;
+        }
+    </style>
+</head>
+
+<body>
+    <!-- The canvas element -->
+    <canvas id="application-canvas"></canvas>
+
+    <!-- The script -->
+    <script>
+        var canvas = document.getElementById("application-canvas");
+
+        // Create the app and start the update loop
+        var app = new pc.Application(canvas);
+
+    </script>
+
+    <!-- LUT table for area lights, included after the app has been created -->
+    <script src="../assets/scripts/lights/area-light-lut.js"></script>
+
+    <script>
+
+        // helper function to create a primitive with shape type, position, scale, color
+        function createPrimitive(primitiveType, position, scale, color, assetManifest) {
+
+            // create material of specified color
+            var material = new pc.StandardMaterial();
+            material.diffuse = color;
+            material.shininess = 80;
+            material.useMetalness = true;
+
+            if (assetManifest) {
+                material.diffuseMap = assetManifest[0].asset.resource;
+                material.normalMap = assetManifest[1].asset.resource;
+                material.glossMap = assetManifest[2].asset.resource;
+                material.metalness = 0.7;
+            }
+
+            material.update();
+
+            // create primitive
+            var primitive = new pc.Entity();
+            primitive.addComponent('model', {
+                type: primitiveType
+            });
+            primitive.model.material = material;
+
+            // set position and scale and add it to scene
+            primitive.setLocalPosition(position);
+            primitive.setLocalScale(scale);
+            app.root.addChild(primitive);
+
+            return primitive;
+        }
+
+        // helper function to create area light including its visual representation in the world
+        function createAreaLight(type, shape, position, scale, color, intensity, shadows, range) {
+            var lightParent = new pc.Entity();
+            lightParent.translate(position);
+            app.root.addChild(lightParent);
+
+            var light = new pc.Entity();
+            light.addComponent("light", {
+                type: type,
+                shape: shape,
+                color: color,
+                intensity: intensity,
+                falloffMode: pc.LIGHTFALLOFF_INVERSESQUARED,
+                range: range,
+                castShadows: shadows,
+                innerConeAngle: 80,
+                outerConeAngle: 85,
+                shadowBias: 0.1,
+                normalOffsetBias: 0.1,
+                shadowResolution: 2048                
+            });
+            
+            light.setLocalScale( scale, scale, scale);
+            light.setLocalEulerAngles( (type === "directional") ? 90 : 0, 0, 0);
```

Comment:
just curious about why this is needed here?

Reply:
it wasn't actually needed - was a overhang from using lookAt() - removed